app/views/timer_sessions/_timer_sessions_day_block.html.slim

```diff
@@ -4,5 +4,7 @@
     b.float-right = sum_work_hours(Array.wrap(timer_sessions))
   table.list.timer-sessions-table
     tbody
-      - Array.wrap(timer_sessions).each do |timer_session|
-        = render 'timer_session_element', timer_session: timer_session
+      - timer_sessions.each do |timer_session|
+        = render TimerSessionEntryComponent.new(timer_session_entry: timer_session,
```

Comment:
With a partial you would not have to pass the variables explicitly.

Reply:
You can do the same with a collection of ViewComponents 😉 TimerSessionEntryComponent.with_collection

Reply:
Yes but you have to pass `@permission_manager` and `@non_matching_timer_sessions[timer_session.id]` explicitly.